platform/android/export/export.cpp

```diff
@@ -1259,6 +1275,31 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		//printf("end\n");
 	}
 
+	bool _icon_file_to_data(const String &p_file_path, Vector<uint8_t> &p_data) {
+		if (p_file_path != "" && p_file_path.ends_with(".png")) {
+			FileAccess *f = FileAccess::open(p_file_path, FileAccess::READ);
+			if (f) {
+				p_data.resize(f->get_len());
+				f->get_buffer(p_data.ptrw(), p_data.size());
+				memdelete(f);
+				return true;
+			}
+		}
+		return false;
+	}
+
+	void _process_launcher_icons(const Ref<EditorExportPreset> &p_preset, const String &p_processing_file_name, const LauncherIcon &p_launcher_icon, Vector<uint8_t> &p_data, const Vector<String> &p_fallback_icons) {
+		if (p_processing_file_name == p_launcher_icon.export_path) {
+			bool success = _icon_file_to_data(String(p_preset->get(p_launcher_icon.option_id)).strip_edges(), p_data);
+
+			if (!success) {
+				for (int i = 0; i < p_fallback_icons.size(); ++i) {
+					_icon_file_to_data(p_fallback_icons[i], p_data);
+				}
```

Comment:
Why is a `for` loop and multiple fallback icons loads needed in the unsuccessful case, when only one icon is loaded in the successful case?

Reply:
That's a fail by me. It should stop the loop as soon as one fallback is found.